docs/5g-on-helium/5g-poc.mdx

```diff
@@ -165,16 +166,28 @@ Network switches that are advertised as "100/10" switches are often "90/9" in re
 
 :::
 
-### Genesis Phase 4: Reward Frequency Increase
+### Genesis Phase 4: Heartbeats and Reliability of Service
 
-> - Reward Periods shortened to 2 Epochs (60 blocks, or approximately 1 hour).
-> - Rewards transactions are triggered with every other Consensus Election.
+> - Reward Period is approximately 1440 blocks and closely aligns with 00:00 UTC to 23:59 UTC.
+> - Reward submission occurs approx 15 minutes after the Reward Period ends, around 00:00 UTC.
+> - 5G Hotspots perform periodic speed tests which will be considered in reward calculations.
+> - 5G Hotspots must generate 1 "heartbeat" in a 1 hour period for at least 12 periods to ensure it
```

Comment:
Small Cell Radios must generate at least 1 "heartbeat" an 1 hour for any 12 hours in the 24-hour reward period.